simplify ways to train so it's more seamless with find search (#431)
DFE-Digital/get-into-teaching-content: content/ways-to-train.md

---

> title: "How to train"

Reviewer: love that we have changed to a question word 'how', think we should reframe a lot of our nav headings like this! Nice

> title: "How to train"

Reviewer 2: What's the rationale for this change?

Author: to make it feel like you have fewer decisions to make - make it more straightforward - to make it more active

  - Assessment-only
  - AO
  - Providers

---

> ## Train to get qualified teacher status

Reviewer: do you think we should capitalise 'Qualified Teacher Status'? As it's an actual qualification name

Author: gov.uk use lowercase - for example: https://www.gov.uk/government/collections/qualified-teacher-status-qts - and policy inform me that "it's not technically a qualification"

> You need ‘qualified teacher status’ to work in the majority of schools in England including state maintained primary, secondary and special schools.

Reviewer 3: Do we need to show the initialism here, as its the first time QTS is mentioned on this page?

```suggestion
You need ‘qualified teacher status’ (QTS) to work in the majority of schools in England including state maintained primary, secondary and special schools.
```

Author: I think I've actually spelled it out across the whole page so didn't bother with the acronym. However, people may see 'QTS' elsewhere so perhaps worth including.

> There are different ways to get qualified teacher status, for example:

Reviewer 2: We have treated jump links very differently on other pages - this list demotes the jump links and means more reading required to understand where to go? One to discuss with design? E.g. see funding your training for jump links established style.

Author: as I mentioned earlier, I'm very much in need of formatting help for this whole page - have asked Gary for help

> * [if you want to go to university as a postgraduate student](#go-to-university-as-a-postgraduate-student)

Author: ```suggestion
* [if you want to return to university as a postgraduate student](#return-to-university-as-a-postgraduate-student)
```

> * [if you want to return to university as a postgraduate student](#return-to-university-as-a-postgraduate-student)

Reviewer 2: Content design - let's not repeat the same words at the front as people glaze - so you put the repeated parts in the set up, then prioritise the different elements in the list. e.g. repetition of 'if you ...'

Author: I would of course normally do this @rosecatt  - I took the view on this occasion that because these are also links, it would be more accessible for screen reader users, who often jump through link text on a page, to get the full context of what they'll get from each section.

> * [if you want to learn on the job in schools near you](#learn-on-the-job-in-schools-near-you)

Reviewer 2: Misleading - learning on the job implies you're being paid. This is being a student in just the same way as option above, it's just saying you're based in school and spend some time in uni rather than vice versa. There are other options where you can 'learn on the job' ie get paid rather than pay for it so we must not confuse them.

Reviewer 2: PS Does it have to be near you?

Author: Decided to include 'near you' because location is such a huge limiting factor for most trainees.

* [if you do not have a degree](#if-you-do-not-have-a-degree)

* [if you already have teaching experience](#if-you-already-have-teaching-experience)

You can also find specialised programmes [if you're a career changer](#if-youre-a-career-changer) or [if you have a doctorate](#if-you-have-a-doctorate).

Not looking for primary or secondary teacher training? Consider teaching [early years](/early-years-teaching-training) or [further education](/further-education-teacher-training).

> ### Go to university as a postgraduate student

Reviewer 3: Wondering if saying ‘Return’ makes it a bit clearer that this is a postgraduate route?

```suggestion
### Return to university as a postgraduate student
```

Author: Oh I like it

> ### Return to university as a postgraduate student

Reviewer 2: For each of the new options, I think there an argument for structuring them in a consistent way like we have currently. Ie with those repeated headings, e.g. primary vs secondary, cost, length of training. 
I remember Myles said that this had really helped with speed of reading and comprehension and it would be good to have that back. Also it draws out the similarity/differences between the options easily which we might have lost. E.g. the fact of how much each option costs vs whether you get paid is harder to find at the moment and that may add confusion.

> If you already have a degree and want to go to university again, you can do a university-led PGCE or PGDE leading to qualified teacher status.

Reviewer 2: first part of sentence repeats header?

A PGCE or PGDE is an academic qualification that you can use towards masters credits.

You spend about a third of your time in academic learning and the rest in school placements.  

Courses typically take one year but you may be able to study part time.

> Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.

Reviewer 4: Is it worth linking directly to `#bursaries-and-scholarships` here? The first section of `/funding-your-training` is on tuition fees and maintenance loans which do cost.

> Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.

Author: ```suggestion
Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.
```
```suggestion
Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects.
```

> Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.

Author: ```suggestion
Fees are usually around £9,250 but you can get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects.
```

> You can also apply for a [student loan](/funding-your-training).

Reviewer 4: Here' we're linking to `/funding-your-training` twice in two sentences. I think this should link directly to `#tuition-fee and-maintenance-loans`.

> You can also apply for a [student loan](/funding-your-training).

Author: ```suggestion
You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-laons).
```

> You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-laons).

Author: ```suggestion
You can also apply for a [student loan](/funding-your-training#tuition-fee-and-maintenance-loans).
```

<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find university training</a>

### Learn on the job in schools near you

> If you already have a degree and want to spend as much time as possible in practical learning, you can find a training programme led by a school or group of schools.

Reviewer: wonder if it's worth including/explaining the acronym SCITT here? My reasoning is because it does seem to be used in the titles of some of the school providers on Find, so might be useful for the user to know

Author: have also questioned this - whether we should at least reference some of the terms people could come across - but unsure

Reviewer 4: Reference as in a separate glossary somewhere? Or something in the page?

Reviewer 2: I would argue that SCITT needs to be here. It's a phrase that is definitely used, and our 'definitive' role means we need to include it

> You'll only spend about about a fifth of your time in academic learning.

Reviewer 3: 👀 

```suggestion
You’ll only spend about about a fifth of your time in academic learning.
```

Reviewer 4: We have [smart quotes](https://kramdown.gettalong.org/options.html#option-smart-quotes) enabled in Kramdown here so they should be automatically applied when the page is rendered. Here's how the above is rendered in the review app with a dumb quote in the doc 👇🏽 

![Screenshot from 2021-04-26 11-04-26](https://user-images.githubusercontent.com/128088/116065726-367ab180-a67f-11eb-9a32-cfb45b3bc744.png)

Reviewer 3: How does that address my OCD, @peteryates? 🤪 (Good to know!)

Author: wooow you're telling me i don't have to worry about curly quotes on GIT @peteryates ?

> You'll only spend about about a fifth of your time in academic learning.

Reviewer 4: The phrase 'academic learning', and 'practical learning' just above feel like they need defining. Could the language be more descriptive here?

> You'll only spend about about a fifth of your time in academic learning.

Author: ```suggestion
You'll only spend about about a fifth of your time in academic learning (for example, written assignments).
```

> You'll only spend about about a fifth of your time in academic learning (for example, written assignments).

Reviewer 2: I would assume academic learning being in-class (whether virtual or not) time, and written assignments being additional to that in one's own time?

Many courses even give you a PGCE or PGDE - an academic qualification which you can use towards masters credits.

Courses typically take one year but you may be able to study part time.

Although you often have to pay course fees of around £9,250 you can get [funding that you do not have to pay back](/funding-your-training) for certain subjects.

You can also apply for a [student loan](/funding-your-training).

> Some training programmes offer a salary, although these can be harder to find.

Reviewer 2: Can we be clear about why they're hard to find - ie is it about your qualifications or experience, or the fact that they are not publicised or online? If there are ones that offer salaries can we link to them or say what the criteria would be?

<a class="button" href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Find courses near you</a>

#### Teach First

> You can also consider [Teach First](https://www.teachfirst.org.uk/). It's a salaried, 2-year programme leading to a PGDE and qualified teacher status.

Reviewer 2: Rather than 'you can also consider', we could just say that Teachfirst is a salaried, 2-year ...

### If you do not have a degree

> You can get qualified teacher status as part of an undergraduate degree.

Reviewer 2: Is it style/policy to call these 'undergraduate' rather than 'bachelor' degrees? I think elsewhere on site (e.g. TTA rules) we call it bachelor but I'm ignorant on style guide apologies ...

Author: we don't have a style guide yet!

> Qualifications vary depending on the course. For example, you could get qualified teacher status with a BA honours or a BSc honours.

Reviewer 2: is it dependent on the course or the subject?

Courses typically cost £9,250 per year and can take up to 4 years but you can [get funding](https://www.gov.uk/student-finance-calculator).

<a class="button" href="https://digital.ucas.com/search">Search for an undergraduate course</a>

### If you already have teaching experience

If you have a degree and you're an experienced teacher, you may be able to get qualified teacher status through a 12-week 'assessment only' programme.

[Apply directly to assessment only teacher training providers](/assessment-only-providers) and ask them about entry criteria.

> ### If you're a career changer

Reviewer 2: Is being a career changer an internal phrase, or a generally understood one? I hadn't heard it before. Slight nervousness on labelling people rather than talking about their choices? e.g. Other options don't name that person as 'A teacher already' vs 'if you already have teaching experience'

A career change into teaching allows you to use your experience and passion to inspire young people.

[Now Teach](https://nowteach.org.uk/) and [Transition to Teach](https://www.transitiontoteach.co.uk/) can support you with your career change.

### If you have a doctorate

If you have or are about to get a PHD you could consider the [researchers in schools](https://thebrilliantclub.org/researchers-in-schools/) programme.

> If you have a lot of relevant work experience you could even earn a salary.

Reviewer 2: have we lost a link here?

<p class="call-to-action__action">
  <a href="https://www.find-postgraduate-teacher-training.service.gov.uk/">Search for a course near <span>you</span></a>
</p>

#### Teach First

You could also consider [Teach First](https://www.teachfirst.org.uk/). This practical, salaried, 2-year programme leads to a postgraduate diploma in education’ (PGDE), similar to a PGCE, and qualified teacher status.

<p>&nbsp;</p>

### If you do not have a degree yet

You can get qualified teacher status as part of an undergraduate degree course.

Qualifications vary depending on the course you choose. For example, you could get a BA honours degree with QTS, or a BSc honours with QTS.

Courses typically cost £9,250 per year and can take up to 4 years - but you can [get funding](https://www.gov.uk/student-finance-calculator).

<p class="call-to-action__action">
  <a href="https://digital.ucas.com/search">Search for an undergraduate <span>course</span></a>
</p>

<p>&nbsp;</p>

### If you already have teaching experience

If you’re an experienced teacher with a degree, you may be able to get QTS without doing a teacher training programme.

> You can [apply directly to assessment only teacher training providers](https://getintoteaching.education.gov.uk/assessment-only-providers).

Author: ```suggestion
You can [apply directly to assessment only teacher training providers](/assessment-only-providers).
```
